2_Build-Minimum-Viable-Product/README.md

```diff
@@ -1,3 +1,9 @@
+---
+layout: default
+title: Phase 2: Building an MVP
```

Comment:
I think we are using new language for this now. (See the DevHub's agile phases card which references alpha, beta, live). 

I don't want this content to talk to much about the process, but will orient it to focus on what the lab offers during this phase.

Can we call this: "Phase 2: Build a Product"